jsapp/scss/mixins.scss

```diff
@@ -0,0 +1,32 @@
+@use "scss/sizes";
+
+@mixin buttonReset() {
```

Comment:
not sure if we need this after new button component?

Reply:
Right, I think I copied `mixins.scss` from some other branch that used that mixin. Will delete it here :)